react/components/TopMenu.js

```diff
@@ -36,31 +37,37 @@ const TopMenu = ({
         showLogin={showLogin}
       />
       
-      <Border fixed top={96} />
+      <Border fixed top={CONSTANTS.COLLAPSIBLE.TOP} />
 
-      <Collapsible top={96} leanMode={leanMode} >
+      <Collapsible top={CONSTANTS.COLLAPSIBLE.TOP} leanMode={leanMode} >
         <ExtensionPoint id="category-menu" />
       </Collapsible>
 
-      <Spacer height={ 96 + 64 } />
+      <Spacer 
+        height={ desktop 
+          ? CONSTANTS.SPACER.DESKTOP
+          : CONSTANTS.SPACER.MOBILE
+        }
+      />
 
     </Fragment>
   )
 }
 
 TopMenu.propTypes = {
-  linkUrl: PropTypes.string,
-  logoUrl: PropTypes.string,
-  logoTitle: PropTypes.string,
-  showSearchBar: PropTypes.bool,
-  showLogin: PropTypes.bool,
-  leanMode: PropTypes.bool,
-  extraHeaders: PropTypes.node,
+  linkUrl: string,
+  logoUrl: string,
+  logoTitle: string,
+  showSearchBar: bool,
+  showLogin: bool,
+  leanMode: bool,
+  extraHeaders: node,
```

Comment:
Add `runtime` here

Reply:
Also, don't use it like that, use `PropTypes.bool` and so on... When we migrate it to TS, it will conflict with its type system

Reply:
✅